src/main/resources/swagger.api/schemas/d2ir/returnUncirculatedDTO.json

```diff
@@ -0,0 +1,23 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "allOf": [
+    {
+      "$ref": "baseCircRequestDTO.json"
+    }
+  ],
+  "description": "INN-Reach Return Circulation Message Put Request Data",
+  "type": "object",
+  "properties": {
+    "title": {
+      "description": "Item title",
+      "type": "string",
+      "maxLength": 256
+    },
+    "author": {
+      "description": "Author",
+      "type": "string",
+      "maxLength": 256
```

Comment:
there is no restriction on length:
![image](https://user-images.githubusercontent.com/42244720/144393623-0f82be04-fc85-4945-a939-83a7cd45d626.png)

Reply:
fixed